c2corg_api/tests/views/test_waypoint.py

```diff
@@ -660,6 +660,34 @@ def test_post_success(self):
             feed_change.area_ids, [self.area1.document_id]
         )
 
+    def test_post_wrong_geom_type(self):
+        body = {
+            'document_id': 1234,
+            'version': 2345,
+            'geometry': {
+                'document_id': 5678, 'version': 6789,
+                'geom': '{"type": "Point", "coordinates": [635956, 5723604]}',
+                'geom_detail':
+                    '{"type": "Point", "coordinates": [635956, 5723604]}'
+            },
+            'waypoint_type': 'summit',
+            'elevation': 3779,
+            'locales': [{
+                'id': 3456, 'version': 4567,
+                'lang': 'en', 'title': 'Mont Pourri',
+                'access': 'y'}
+            ],
+            'associations': {
+                'waypoint_children': [
+                    {'document_id': self.waypoint2.document_id}
+                ]
+            }
+        }
+        errors = self.post_wrong_geom_type(body)
+        self.assertEqual(
+            errors[0]['description'], "Invalid geometry with expected type: "
+            "['POINT']. Got MULTIPOINT")
```

Comment:
Is this test correct? The type of `geom_detail` is `Point` but the error message says "Got MULTIPOINT"?